public/styles.css

```diff
@@ -119,6 +119,7 @@ table {
 
 .mobile-content {
     padding: .5rem;
+    min-width: 350px;
```

Comment:
Yep ~ feel free to fix this and close this PR if you have simple and elegant solution, I do not have simple css solution right now, need to do some study this later